views/jira-connect-ghe.hbs

```diff
@@ -0,0 +1,103 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="ap-local-base-url" content="{{localBaseUrl}}" />
+    <meta name="public-url" content="{{APP_URL}}" />
+    <title>{{title}}</title>
+    <link
+      href="/public/aui/aui-prototyping.css"
+      rel="stylesheet"
+      integrity="DTM1Q+8lU7SzJT+FWr0JFisCSZlwfM0GiAKYy7h1s9vIKa/CIh37s9NuOCqIOgK4tmqrjLK4NuWuIPUQNsikHA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="/public/css-reset/bundle.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/atlassian-ui-kit/bundle.css"
+      media="all"
+    />
+    <link rel="stylesheet" href="/public/css/global.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/css/jira-connect-ghe.css"
+      media="all"
+    />
+    <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
+  </head>
+
+  <body class="aui-page-hybrid">
+    <div class="jiraConnectGHE">
+      <header class="jiraConnectGHE__header">
+         {{> navigation previousPageName="Select GitHub Version" }}
+      </header>
+
+      <section class="jiraConnectGHE__content">
+        <div class="headerImage">
+          <img
+            src="/public/assets/jira-and-github.png"
+            alt="Jira and GitHub logos"
+          />
+        </div>
+        <h2 class="jiraConnectGHE__content__title">GitHub Enterprise Server</h2>
+
+        <div class="jiraConnectGHE__prompt">
+            Connecting GitHub Enterprise Server to Jira requires opening a hole in your firewall,
+            as Atlassian’s IP addresses must be able to call your GitHub Enterprise Server.
+          <!--TODO: Add a link here-->
+          <a href="" target="_blank">Learn more.</a>
+        </div>
+
+        <div>
+          <h3 class="jiraConnectGHE__options__subTitle">
+            GitHub Enterprise Server URL&nbsp;
+            <!--TODO: Add a tooltip for this icon-->
```

Comment:
Going off Jade's comment in slack, let's remove this todo and the question mark.

Reply:
Removed